CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+## 2.12.0-rc.0
+
+* [FEATURE] Track currently active PromQL queries in a log file. #5794
+* [FEATURE] Enable and provide binaries for `mips64` / `mips64le` architectures. #5792
+* [ENHANCEMENT] Improve responsiveness of targets web UI and API endpoint. #5740
+* [ENHANCEMENT] Improve remote write desired shards calculation. #5763
+* [ENHANCEMENT] Flush TSDB pages more precisely. tsdb#660
+* [ENHANCEMENT] Add `prometheus_tsdb_retention_limit_bytes` metric. tsdb#667
+* [ENHANCEMENT] Add logging during TSDB WAL replay on startup. tsdb#662
+* [ENHANCEMENT] Improve TSDB memory usage. tsdb#653, tsdb#643, tsdb#654, tsdb#642, tsdb#627
+* [BUGFIX] Check for duplicate label names in remote read. #5829
+* [BUGFIX] Mark deleted rule's series as stale on next evaluation. #5759
```

Comment:
rules'

Reply:
Haha I was debating whether I should fix / change that :P Done.